step2-07/demo/README.md

```diff
@@ -47,24 +47,47 @@ const NewComponent = connect(
 
 `connect()` takes in a few functions that map portions of the state tree and dispatcher functions into props. It is a **higher-order function**, meaning that the return value of `connect()` is a function that decorates `OldComponent` into a `NewComponent` with all the mapped props.
 
-A [`mapStateToProps`](https://react-redux.js.org/api/connect#mapstatetoprops-state-ownprops-object) function selects out portions of the state tree. This function informs the connected view when to re-render based on a shallow comparison from previous state.
+#### `mapStateToProps`
+
+A [`mapStateToProps`](https://react-redux.js.org/api/connect#mapstatetoprops-state-ownprops-object) function returns portions of the state tree that will be passed to the component as props.
+
+Under the hood, this function helps inform the connected view when it should re-render based on a shallow comparison with the previous state.
 
 ```ts
-function mapStateToProps(state) {
+interface Store {
```

Comment:
Added the involved interfaces (and some more notes) because I think that makes it clearer what's going on